sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -3,14 +3,9 @@
 if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_DIR)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  #commit b38855ed815ffd076bfde5e5e06170ca4f723dc1
-  #Merge: e6343f4 6a2c548
-  #Author: Piotr Balcer <piotr.balcer@intel.com>
-  #Date:   Thu Oct 5 12:15:42 2023 +0200
-  #    Merge pull request #920 from jsji/localcopy
-  #        [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path
-  set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
+  # Do not merge!!
+  set(UNIFIED_RUNTIME_REPO "https://github.com/hdelan/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG 2b2e2346ebe231f6155d45a67c428b2a84671a86)
```

Now that https://github.com/oneapi-src/unified-runtime/pull/961 is merged please pull in the latest changes from the sycl branch and update this to:

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 036b9cfcd8fb4be9394449b5406e6402580a63c9
  # Merge: 2ab0734 86f96f0
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Fri Oct 27 16:36:23 2023 +0100
  #     Merge pull request #961 from hdelan/change-unions-to-stdvariant
  #     [HIP][CUDA] Change unions in ur_mem_handle_t_ to stdvariant
  set(UNIFIED_RUNTIME_TAG 036b9cfcd8fb4be9394449b5406e6402580a63c9)
```

The PR title/description will need updated to reflect the contained UR changes. Then mark as ready for review.